data-raw/_helper.R

```diff
@@ -0,0 +1,24 @@
+library(here)
+
+internal_data_path <- here("R/sysdata.rda")
+
+save_to_sysdata <- function(obj, obj_name) {
+  # Create a new environment for storing sysdata contents
+  sysdata_env <- new.env(hash = FALSE)
```

Comment:
```suggestion
  sysdata_env <- new.env(hash = FALSE, parent = emptyenv())
```
Just in case, hedging against weird stuff that might happen in some versions/futures.